.github/workflows/pushpull.yaml

```
@@ -16,7 +16,7 @@ jobs:
       - uses: "actions/checkout@v2"
 
       - name: HACS validation
-        uses: "hacs/integration/action@master"
+        uses: "hacs/action"
```

```suggestion
        uses: "hacs/action@main"
```
Branch/tag is required

---

I was just comparing the content with my other custom integration (https://github.com/custom-components/media_player.braviatv_psk/blob/master/.github/workflows/pushpull.yaml) so will make some more updates to this file, as I have updated the bravia integration more recently.